app/templates/components/project-form.hbs

```diff
@@ -31,18 +31,21 @@
         mode="developmentSite"
         modeDisplayName="Development Site"
         hasGeom=model.developmentSite
+        geometryMode=geometryMode
```

Comment:
Since `project-form` is a contextual component, can you just pass `geometryMode` through the `project-geometries-menu`'s `{{yield}}`? This is how the other repeatable properties get passed. See https://github.com/NYCPlanning/labs-applicantmaps/blob/develop/app/templates/components/project-geometries-menu.hbs#L3-L7

Reply:
Fixed in 0674cdce1220fc8edbb371fc4da4801656d29eaa